common/lib/capa/capa/templates/chemicalequationinput.html

```diff
@@ -1,9 +1,10 @@
+<%! from openedx.core.djangolib.markup import Text %>
 <div id="chemicalequationinput_${id}" class="chemicalequationinput">
     <div class="script_placeholder" data-src="${previewer}"/>
 
     <div class="${status.classname}" id="status_${id}">
 
-    <input type="text" name="input_${id}" id="input_${id}" aria-label="${response_data['label']}" aria-describedby="answer_${id}" data-input-id="${id}" value="${value|h}"
+    <input type="text" name="input_${id}" id="input_${id}" aria-label="${Text(response_data['label'])}" aria-describedby="answer_${id}" data-input-id="${id}" value="${value|h}"
```

Comment:
@robrap Here we are escaping the `response_data['label']` because it may contain HTML.